src/fetchers/top-languages-fetcher.js

```diff
@@ -6,10 +6,10 @@ const fetcher = (variables, token) => {
   return request(
     {
       query: `
-      query userInfo($login: String!) {
+      query userInfo($login: String!, $ownerAffiliations: [RepositoryAffiliation]) {
         user(login: $login) {
-          # fetch only owner repos & not forks
-          repositories(ownerAffiliations: OWNER, isFork: false, first: 100) {
+          # do not fetch forks
+          repositories(ownerAffiliations: $ownerAffiliations, isFork: false, first: 100) {
```

Comment:
Because of https://github.com/anuraghazra/github-readme-stats/pull/1122#issuecomment-1152066225 I think for now we should change this to:

```suggestion
          repositories(ownerAffiliations: $ownerAffiliations, isFork: false, first: 100) {
          repositories(ownerAffiliations: OWNER, isFork: false, first: 100) {
```